.travis.yml

```diff
@@ -10,10 +10,21 @@ install:
   - "pip install -r dev-requirements.txt"
   - pip install python-coveralls
   - pip install coverage
+before_script:
+  # We use before_script to report version and path information in a way
+  # that can be easily hidden by Travis' log folding. Moreover, a nonzero
+  # exit code from this block kills the entire job, meaning that if we can't
+  # even sensibly get version information, we correctly abort.
+  - which python
+  - python --version
+  - which nosetests
+  - nosetests --version
+  - which pylint
+  - pylint --version
 script:
   - nosetests --with-coverage -w instruments
-  - pylint --py3k instruments/
-  - pylint instruments/
+  - pylint --py3k instruments
+  - pylint --disable=I instruments
```

Comment:
good call, should help a lot. I wonder if there's a way to put that in the `.pylintrc` file to make it the default. I'm good with this though